tap/tracker.py

```diff
@@ -36,6 +36,9 @@ def __init__(
         # Stream output directly to a stream instead of file output.
         self.streaming = streaming
         self.stream = stream
+        # the plan is how many tests we expect (or None if we don't know yet)
```

Comment:
To keep a consistent style with the rest of the code, could you use proper sentence structure (i.e., capitalization and periods)?